database/book.js

```diff
@@ -0,0 +1,20 @@
+import { Decimal128 } from "mongodb";
```

Comment:
Good that u keept your books schema on a page, did keps the server.js more clean